src/config_entries.c

```diff
@@ -67,6 +67,36 @@ int git_config_entries_new(git_config_entries **out)
 	return error;
 }
 
+int git_config_entries_dup_entry(git_config_entries *entries, const git_config_entry *entry)
+{
+	git_config_entry *dup;
```

Comment:
I think that *could* induce cringing from special-snowflake compilers ? A point for the C90-compatible discussion maybe ?

Reply:
Good catch, I've renamed it to `entry` instead